torchvision/datasets/hmdb51.py

```diff
@@ -64,7 +65,17 @@ def __init__(self, root, annotation_path, frames_per_clip, step_between_clips=1,
         self.samples = make_dataset(self.root, class_to_idx, extensions, is_valid_file=None)
         self.classes = classes
         video_list = [x[0] for x in self.samples]
-        video_clips = VideoClips(video_list, frames_per_clip, step_between_clips)
+        video_clips = VideoClips(
+            video_list,
+            frames_per_clip,
+            step_between_clips,
+            frame_rate,
+            precomputed_metadata,
+            precomputed_metadata_filepath,
```

Comment:
I'm not sure it's a good idea to have the API supporting `precomputed_metadata` and `precomputed_metadata_filepath`, I think that it should support only one of the use-cases, which can be obtained by the other easily. But this is unrelated to this PR, and is dependent on a different PR, so let's move this part of the discussion elsewhere